README.md

```diff
@@ -1077,6 +1133,25 @@ zi as'null' lucid sbin wait'1' for \
 
 Target directory for installed files is `$ZPFX` (`~/.local/share/zinit/polaris` by default).
 
+### Named directories for frequently used paths
+
+Named directories are shorthands in the form of `~NAMED_DIR`
+(upper case not required). Zinit provides a set of such
+names for its frequently used paths:
```

Comment:
As someone who never heard of this feature, an example would be nice? Can I just use `cd ~ZIPLUGS` to be in zinits plugin dir? Unfortunately, `cd ~ZIPLUGS/<tab>` didn't bring out the proper plugin :-(

I also miss one for the git dir, where the zinit source is in